run_sparkle_test.py

```
@@ -0,0 +1,43 @@
+exec(open('{test_script}').read())
+"""],
+                capture_output=True,
+                text=True
+            )
+            return result.stdout + result.stderr, result.returncode
+        except Exception as e:
+            return f"ERROR running tests: {e}", 1
+
+if __name__ == '__main__':
+    # Run tests on cleanup_final
+    worktree = '/tmp/cleanup_final'
```

Ruff (0.16.3)

[error] Probable insecure usage of temporary file or directory: "/tmp/cleanup_final"

(S108)